build.sbt

```diff
@@ -26,6 +26,10 @@ libraryDependencies += "org.testng" % "testng" % "6.8.8"
 
 unmanagedBase := baseDirectory.value
 
+mappings in (Compile, packageBin) ++= Seq(
+  (baseDirectory.value / "lib/jni/libIntelDeflater.so") -> "lib/jni/libIntelDeflator.so"
+)
+
```

Comment:
Everything is unchanged if we're building with ant.  It will continue to do whatever it already does.